.travis.yml

```diff
@@ -1,28 +1,36 @@
 language: python
+
 python:
 - '3.6'
 - '3.5'
 - '3.4'
 
 matrix:
   include:
-    - python: 3.6
-      sudo: required
-      dist: trusty
+    - python: '3.4'
       env:
-      - TOXENV=style
+      - DNF=yum OS=centos7
```

Comment:
Sorry one more thing.
How about moving below line to common `before_install` section?
Because we can select docker image with $CONTAINER_IMAGE variable easily.

```
before_install:
  - if [ "${${CONTAINER_IMAGE}" != '' ]; then
        sed -i "s@fedora@${CONTAINER_IMAGE}@" Dockerfile
    if
```